hack/build

```
@@ -0,0 +1,27 @@
+#!/usr/bin/env -S dagger shell --no-mod
+
+# HACK: strip "build" from the script path to get the parent module
+.use ${0%/build}/..
+
+CONTAINER=${_EXPERIMENTAL_DAGGER_DEV_CONTAINER:-dagger-engine.dev}
+IMAGE=${_EXPERIMENTAL_DAGGER_DEV_IMAGE-}
+
+cli |\
+    dev-binaries --runner-host=docker-container://$CONTAINER --platform=current |\
+    export ./bin &
+
+engine |
+    container ${_EXPERIMENTAL_DAGGER_GPU_SUPPORT:+--gpu-support} |\
+    as-tarball --forced-compression=gzip |\
+    export ./bin/engine.tar &
+engine |\
+    load-to-docker ${DOCKER_HOST:-/var/run/docker.sock} \
```

```suggestion
    load-to-docker ${DOCKER_HOST:-$(docker context inspect | jq -r '.[0].Endpoints.docker.Host')} \
```

/var/run/docker.sock is fairly linux-specific i think? definitely doesn't work by default with colima.

---

i think this works too and doesn't require jq:

```suggestion
    load-to-docker ${DOCKER_HOST:-$(docker context inspect -f '{{.Endpoints.docker.Host}}')} \
```

---

Yeah, we can't put bash commands in dagger shell - intentionally, it's intended that there's no way to escape the sandbox :cry:

I can put this logic into `./hack/dev` though.

---

This *feels* related to https://github.com/dagger/dagger/issues/8025

---

huh @helderco this looks like a bug :eyes: